cmd/node/config/enableEpochs.toml

```diff
@@ -242,22 +242,23 @@
     # StakeLimitsEnableEpoch represents the epoch when stake limits on validators are enabled
     StakeLimitsEnableEpoch = 5
 
-    # StakingV4InitEnableEpoch represents the epoch when staking v4 is initialized. This is the epoch in which
+    # StakingV4Step1EnableEpoch represents the epoch when staking v4 is initialized. This is the epoch in which
     # all nodes from staking queue are moved in the auction list
-    StakingV4InitEnableEpoch = 4
+    StakingV4Step1EnableEpoch = 4
 
-    # StakingV4EnableEpoch represents the epoch when staking v4 is enabled. Should have a greater value than StakingV4InitEnableEpoch
-    StakingV4EnableEpoch = 5
+    # StakingV4Step2EnableEpoch represents the epoch when staking v4 is enabled. Should have a greater value than StakingV4Step1EnableEpoch.
+    # From this epoch, all shuffled out nodes are moved to auction nodes. No auction nodes selection is done yet.
+    StakingV4Step2EnableEpoch = 5
```

Comment:
do we have protection at the code level not to mistakenly put a lower or equal value than StakingV4Step1EnableEpoch? Valid also for the StakingV4Step3EnableEpoch param. If not, it can be made in a new PR.

Reply:
No, there is no check yet.
Added a task for it

Reply:
great!